app/controllers/concerns/omni_concern.rb

```diff
@@ -0,0 +1,89 @@
+module OmniConcern
+  extend ActiveSupport::Concern
+
+  def create
+    @auth = request.env["omniauth.auth"]
+    assign_credential
+    if (!current_account.nil?)
```

Comment:
Rubocop is complaining about this line. Can you instead say `if current_account.present?` ?